lmnet/lmnet/datasets/base.py

```diff
@@ -95,7 +95,7 @@ def __getitem__(self, i, type=None):
     @property
     @abstractmethod
     def __len__(self):
-        """returns the number of items in the dataset."""
+        """returns the number of datas in the dataset."""
```

Comment:
Original explanation is correct, could you revert this line?

BTW, `data` is already plural form, hence we cannot say `datas`. (This means that line 86 is wrong... 😢)

Reply:
I revert this, done. :)